test/integration/stamp_editor_spec.mjs

```diff
@@ -698,4 +699,67 @@ describe("Stamp Editor", () => {
       );
     });
   });
+
+  describe("Resize a stamp", () => {
+    let pages;
+
+    beforeAll(async () => {
+      pages = await loadAndWait("tracemonkey.pdf", ".annotationEditorLayer");
+    });
+
+    afterAll(async () => {
+      await closePages(pages);
+    });
+
+    it("must check that a stamp can be undone", async () => {
```

Comment:
Shouldn't this be "must check if a stamp can be resized", or something similar (I presume this was copy/pasted)?